contributing/releasing.md

```diff
@@ -19,6 +19,17 @@ is expected to do the following, in order of priority:
 If something is stopping the release engineer from achieving any of the above goals, the culprit
 code should be removed immediately from the release.
 
+If you are not able to cut a release by Wednesday afternoon this is *cause for concern*. Remember:
+landing the release is more important than landing any individual change. Revert suspect changes
+aggressively if something might be holding back the release.
+
+If you are going to be unavailable on the next working day and the release has not yet landed by
+Wednesday afternoon, you must hand the release off to the next release engineer before the end
+of the working day.
+
+Never land a release after 3pm New York time on any work day. If the release has to roll back then the
```

Comment:
If you are blocking it after 3pm then the likelihood of a release starting and going out on Wedensday is low.

Reply:
Moving discussion to the thread above.

Reply:
GitHub keeps hiding the other thread because it's "outdated" :(

What should we do here? Agreed that 3pm can be tough for a release to land by, though it should be doable.

Reply:
What is "pulling the trigger" in the release process? Starting testing?

I think it's always been OK for releases to land on Thursdays. Did we change this recently?

Reply:
pulling the trigger = submitting CL (after making sure the day before everything is good), publishing release on GitHub and CocoaPods.